pika/connection.py

```diff
@@ -336,7 +337,8 @@ def __init__(self,
                  retry_delay=None,
                  socket_timeout=None,
                  locale=None,
-                 backpressure_detection=None):
+                 backpressure_detection=None,
+                 client_props=None):
```

Comment:
What about integration with the `Parameters` class, as is the case for all other args, including `DEFAULT_*` and `_validate_*`?